packages/frames.js/src/middleware/images-worker/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./imagesWorker";
```

Comment:
why this file doesn't contain the contents of the exported file?

Reply:
what do you mean?

Reply:
fixed